supervision/keypoint/annotate.py

```diff
@@ -0,0 +1,104 @@
+from abc import ABC, abstractmethod
+from typing import Optional
+
+import cv2
+import numpy as np
+
+from supervision.annotators.base import ImageType
+from supervision.draw.color import Color
+from supervision.keypoint.core import KeyPoints
+from supervision.keypoint.skeletons import KnownSkeletons, Skeleton
+from supervision.utils.conversion import convert_for_annotation_method
+
+
+class BaseKeyPointAnnotator(ABC):
+    @abstractmethod
+    def annotate(self, scene: ImageType, detections: KeyPoints) -> ImageType:
+        pass
+
+
+class VertexAnnotator(BaseKeyPointAnnotator):
+    def __init__(
+        self,
+        color: Color = Color.ROBOFLOW,
+        radius: int = 4,
+    ) -> None:
+        """
+        Most basic keypoint annotator.
+
+        Args:
+            color (Color, optional): The color of the keypoint.
+            radius (int, optional): The radius of the keypoint.
+        """
+        self.color = color
+        self.radius = radius
+
+    @convert_for_annotation_method
+    def annotate(self, scene: ImageType, keypoints: KeyPoints) -> ImageType:
+        if len(keypoints) == 0:
+            return scene
+
+        xy_all = keypoints.xy
+        for xy in xy_all:
+            for x, y in xy:
+                cv2.circle(
+                    img=scene,
+                    center=(int(x), int(y)),
+                    radius=self.radius,
+                    color=self.color.as_bgr(),
+                    thickness=-1,
+                )
+
+        return scene
+
+
+class SkeletonAnnotator(BaseKeyPointAnnotator):
+    def __init__(
+        self,
+        color: Color = Color.ROBOFLOW,
+        thickness: int = 2,
+        skeleton: Optional[Skeleton] = None,
+    ) -> None:
+        """
+        Draw the lines between points of the image.
+
+        Args:
+            color (Color, optional): The color of the lines.
+            thickness (int, optional): The thickness of the lines.
+            skeleton (Skeleton, optional): The skeleton to draw.
+                If set to `None`, will attempt to select automatically.
+        """
+        self.color = color
+        self.thickness = thickness
+        self.skeleton = skeleton
+
+    @convert_for_annotation_method
+    def annotate(self, scene: ImageType, keypoints: KeyPoints) -> ImageType:
+        if len(keypoints) == 0:
+            return scene
+        if keypoints.class_id is None:
```

Comment:
Do we need `class_id` at this point?

Reply:
I have no use for it, and I doubt anyone would need it in this form.

Reply:
Let's drop it for now.